### PR TITLE
Add Release note for 2.15.0

### PR DESCRIPTION
**Pull request by zhyuanqi**
### Description

Add Release note for 2.15.0 using https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2732

## Changelog
- skip

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by BionIT — APPROVED** [review]

File: release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### 🐛 Bug Fixes
+* add http://www.site.com to lycheeignore in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6781
+* [Workspace] Fix: Show a error toast when workspace read only user delete saved objects in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6783
+* [Workspace] Fix workspace name duplication check in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6813
+* Remove unused import and property (#6879) in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6884
+* fix for quickrange to use datemath to parse datetime strings in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6834
+* [BUG] fix default data source bug in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6912
+* [Discover][Bug] Migrate global state from legacy URL in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6909
+* Revert 815d2bd7c612450e2a15a6543c7e74558adf4a81 "Feat (core): Make theme settings user-specific and user-configurable (#5652) (#6681)" in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6978
```
Comment (ananzh) [inline release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md:29]: I think we could exclude this one

File: release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+## What's Changed
```
Comment (ananzh) [inline release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md:1]: Missing PRs
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6613
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6721
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6759
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6811
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6851
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6777

**Review by zhongnansu — COMMENTED** [review]
missing PRs:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6879
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6851
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6822
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6755
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6729
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6715
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6678
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385

File: release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md (hunk 2 above)
Comment (zhyuanqi) [reply release-notes/release-notes/opensearch-dashboards.release-notes-2.15.0.md:1]: Regenerated the doc

**Review by zhongnansu — APPROVED** [review]

**Review by BionIT — APPROVED** [review]

**Pull request merged**